app/src/main/java/com/example/musicdemomvp/local/asyncTask/OnDataLoadCallBack.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.musicdemomvp.local.asyncTask
+
+interface OnDataLoadCallBack<T> {
+    fun onSuccess(data: T?)
+    fun onFail(e: String)
```

Comment:
đặt tên biến rõ ra e nhé 
```suggestion
    fun onFail(error: String)
```